app/components/course-page/previous-steps-incomplete-modal.hbs

```diff
@@ -1,29 +1,35 @@
 <ModalBody @allowManualClose={{false}} @onClose={{@onClose}} class="w-full" data-test-previous-steps-incomplete-modal ...attributes>
-  <div class="mb-4 font-semibold text-2xl text-gray-800 dark:text-gray-100 mr-6">
-    Previous steps incomplete
-  </div>
+  <div class="flex flex-col items-center w-full">
+    <div class="text-[60px] text-center">
+      ⚠️
+    </div>
 
-  <div class="prose dark:prose-invert mb-6">
-    <p>
-      This step depends on previous steps that you haven't completed yet.
-    </p>
-  </div>
+    <div class="mb-1 font-bold text-2xl text-gray-700 dark:text-gray-100 text-center">
+      Previous steps incomplete!
+    </div>
+
+    <div class="prose dark:prose-invert mb-5 text-center" data-test-completion-message>
+      <p>
+        This step depends on previous steps<br />
```

Comment:
```suggestion
        This step depends on previous steps
```

Let's not force wrapping like this, it'll look weird on certain screen sizes. If you just want it to look balanced, maybe try [this](https://tailwindcss.com/docs/text-wrap#balanced-text-wrapping)?